src/db/sysdb_subdomains.c

```diff
@@ -172,6 +184,8 @@ struct sss_domain_info *new_subdomain(TALLOC_CTX *mem_ctx,
     dom->homedir_substr = parent->homedir_substr;
     dom->override_gid = parent->override_gid;
 
+    dom->gssapi_services = parent->gssapi_services;
+
```

Comment:
Do you think it would make sense to check this option in `check_subdom_config_file()` as well?

Reply:
Done.